node/src/BaseClient.ts

```diff
@@ -1123,6 +1123,10 @@ export class BaseClient {
                   batch: command_request.Batch.create({
                       isAtomic: true,
                       commands: command,
+                      // TODO: add support for timeout
+                      raiseOnError: true,
```

Comment:
You can make these fields in protobuf optional so you won't need to pass these values